File: lab3/bmicalc/app/src/main/java/eu/xthauron/bmicalc/Model/Question.java
package eu.xthauron.bmicalc.Model;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Question implements Serializable {
    private final static long serialVersionUID = 8351897667222850048L;

    private String name;
    private List<Answer> answers = new ArrayList<>();
    private int correctAnswerId;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Answer> getAnswers() {
        return answers;
    }

    public void setAnswers(List<Answer> answers) {
        this.answers = answers;
    }

    public int getCorrectAnswerId() {
        return correctAnswerId;
    }

    public void setCorrectAnswerId(int correctAnswerId) {
        this.correctAnswerId = correctAnswerId;
    }
}

File: lab4/BmiCalc2/app/src/main/java/eu/xthauron/bmicalc/Model/Question.kt
package eu.xthauron.bmicalc.Model

import java.io.Serializable
import java.util.*

class Question : Serializable {
    var name: String? = null
    var answers: List<Answer> = ArrayList()
    var correctAnswerId = 0

    companion object {
        private const val serialVersionUID = 8351897667222850048L
    }
}